deno/lib/README.md

```diff
@@ -57,8 +57,8 @@
 - [Coercion for primitives](#coercion-for-primitives)
 - [Literals](#literals)
 - [Strings](#strings)
-  - [Datetime](#datetime-validation)
-  - [IP](#ip-address-validation)
+  - [Datetime](#iso-datetimes)
+  - [IP](#ip-addresses)
```

Comment:
Can we move all of the changes in this deno README to a separate PR if they're relevant?

Reply:
I have not modified this file, the possible reason is that the build automatically generates.

Reply:
I re-committed again and removed the changed code. But there is no diff.

Reply:
@scotttrinh

Reply:
@recallwei 

I think the issue is that you need to merge `origin:master` into your branch and try again.

Reply:
@scotttrinh Ok, I synced the master branch.